style.css

```diff
@@ -0,0 +1,3 @@
+.h1 {
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure your class has always a descriptive name. 
Hint: `header or title` would make more sense instead of `h1` :smile: